### DISCUSSION TITLE
Support study groups

### DISCUSSION THREAD

**Discussion by s-leroux** [EddieHubCommunity/support#3980] [Ideas]
This idea has raised from a Discord discussion:
* https://discord.com/channels/699608417039286293/699608417928740944/977114704851922944
* https://discord.com/channels/699608417039286293/699608417928740944/977115661828521984
* https://discord.com/channels/699608417039286293/699608417928740944/977117257970884608

Many of us are taking online courses. Often we take the same courses but alone on our side. It would be great if the members of EddieHub could form study groups to follow those courses together. Ideally, mixing people with different expertise on the topic, so the most advanced ones could serve as mentors for the students with less experience.

We may imagine taking the lectures or doing the homework at the same time, using a dedicated chat room to discuss the course or the issues encountered.

Question: How to deal with participants from different time zones?

Option: I don't know if this is in the scope of EddieHub, but we may also imagine developing partnerships with online course platforms to obtain group rates.

**Comment by ewainy** [comment]
I just came here to say that I really love this idea and would love to see this happen!! 💡
As for timezones, perhaps the weekend might be the best chance for everyone to be able to participate. Maybe a mix of a bigger group at the weekend and small groups with similar timezones during the week?
What would also be great is if we had a repo for people to add their notes and reflections, and areas where they need more help and then there could be some tailored mentoring arranged too 🤔 or we could practice building a project together with what we're learning.

Anyway, can't wait to see everyone's thoughts on this 💖

**Comment by s-leroux** [comment]
> we could practice building a project together with what we're learning.

I had something similar in mind. That would be the occasion to work collaboratively on a limited scope project we could showcase on our resume or {LinkedIn, GitHub, whatever} profile.

> What would also be great is if we had a repo for people to add their notes and reflections, and areas where they need more help and then there could be some tailored mentoring arranged too

Can't we leverage the "discussion" section of the aforementioned project for that?

**Reply by ewainy** [reply]
I agree! Would be great! 😊

Ah yes, I like the idea of leveraging discussions for opinions or asking for help
But I'm a huge fan of documenting in Markdown because we could format that into a small group learning blog and then that's another link/project evidence and more opportunities for people to work on open source, reviewing eachothers pull requests

**Comment by EmmaDawsonDev** [comment]
I'm not sure if fixed study groups is the way to go; it can be hard to manage groups of people spread over different time zones with different time commitments and people wanting to go at their own learning pace. However, I think having study sessions together could be worthwhile, something like Eddie is already planning with the Centered App, or having some sort of time on the calendar each week for discussions and support.

**Reply by s-leroux** [reply]
It seems two different things emerged from the discussion:

1. The idea of a study group per se where students exchange together on the content of the course and homework. It may be addressed by a weekly discussion and support session for example
2. The idea of building collaboratively on top of what was studied. For now we have two (possibly complementary) tracks: (2a) building an OSS end of study project based on what was learned. This project would focus on applying the newly learn technology (2b) having a repo to take notes and review them during the course of the studies. This project would focus more on communication, technical writing (and English writing!)

About leveraging the Centered App for that, I didn't follow the discussion so I can't tell much about that eventuality.

**Comment by eddiejaoude** [comment]
Great idea and discussion everyone 🔥 

Here are some of my thoughts:
- I do see some challenges with regarding to the course, each person would have to buy the same course and go at the same rate through the course - in addition to timezones, everyone has hours they can commit to learning depending on work/family etc
- Building a project together is always great, but isn't this just open source? 
- We do have the Centered group which is half way there, but everyone works on their own thing that is unrelated, but encourages each other to focus and not get distracted

**Comment by LR-TechX** [comment]
A study session is a great idea for me, specially for online courses but another way around it would be to put out key points suggested from the studies and anyone can identify faults from there and discuss... That would spark up the liveliness 🤝🏻🥂
